mipengine/common/validate_identifier_names.py

```diff
@@ -3,30 +3,43 @@
 
 
 def validate_identifier_names(func):
-    wraps(func)
-
+    @wraps(func)
```

Comment:
As Jason said on the other PR, there should be one PR with changes on a specific component. 

It's much easier to review targeted PRs.

If you need help moving commits from one branch to another let me know.